check_line_length.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import glob
+import os
+import re
+import sys
+
+LIMIT = 75
+
+
+def check_url(line):
+    """
+    Check if there is a url present in the given line
+    """
+
+    url = \
+        re.findall('http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\), ]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+                   , line)
+    if url:
+        return True
+    else:
+        return False
+
+
+def limit_line_length(file_path):
+    """
+    Check if length of each line of a given file is in the given limit
+    """
+
+    file = file_path.split('/')[-1]
+
+    with open(file_path) as f:
+        lines = f.readlines()
+
```

Comment:
remove blank line (increase line-height of your editor if you feel the need to do this)